ledger/indexer-benchmark/src/main/scala/ledger/indexerbenchmark/Config.scala

```diff
@@ -72,16 +72,6 @@ object Config {
         .action((value, config) =>
           config.copy(indexerConfig = config.indexerConfig.copy(submissionBatchSize = value))
         )
-      opt[Int]("indexer-tailing-rate-limit-per-second")
```

Comment:
are we sure we want to do all these config changes together here? This is breaking, no?
previously we tried to batch the breaking changes, so we have a batch effort to maintain vmbc and LR.
@mziolekda WDYT?

Reply:
(...and keep the configuration, just making it idle)

Reply:
I don't think these are manually configured in any deployment. But @mziolekda can confirm probably.

Reply:
These parameters aren't used in our tests atm, so it is safe to kill